### PR TITLE
Note in the README that we are moving to timed releases

### PR DESCRIPTION
**Pull request by mheon**
I'll also throw this on Podman.io and maybe write a short blog about it there.

#### Does this PR introduce a user-facing change?

```release-note
Podman is changing its release model. We will now make 4 releases a year, once each in February, May, August, and November.
```

### CODE REVIEW

**Review by giuseppe — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -11,8 +11,7 @@ Podman (the POD MANager) is a tool for managing containers and images, volumes m
 Podman runs containers on Linux, but can also be used on Mac and Windows systems using a Podman-managed virtual machine.
 Podman is based on libpod, a library for container lifecycle management that is also contained in this repository. The libpod library provides APIs for managing containers, pods, container images, and volumes.
 
-
-All releases are PGP signed. Public keys of members of the team approved to make releases are located [here](https://github.com/containers/release-keys/tree/main/podman).
+Podman releases 4 times a year, during the second week of February, May, August, and November. All releases are PGP signed. Public keys of members of the team approved to make releases are located [here](https://github.com/containers/release-keys/tree/main/podman).
```
[the remark this answers is not in the document]
Comment (mheon) [reply README.md:14]: Yes, fair.

[the remark this answers is not in the document]
Comment (mheon) [reply README.md]: Added another sentence about this.

**Review by vrothberg — APPROVED** [review]
LGTM

**Review by Luap99 — APPROVED** [review]
LGTM

**Pull request merged**